Source/WebCore/dom/Node.cpp

```diff
@@ -2477,8 +2477,10 @@ static inline bool tryAddEventListener(Node* targetNode, const AtomString& event
     if (typeInfo.isInCategory(EventCategory::Wheel)) {
         document->didAddWheelEventHandler(*targetNode);
         document->invalidateEventListenerRegions();
-    } else if (isTouchRelatedEventType(typeInfo, *targetNode))
+    } else if (isTouchRelatedEventType(typeInfo, *targetNode)) {
         document->didAddTouchEventHandler(*targetNode);
+        document->invalidateEventListenerRegions();
```

Comment:
Can we have `Document:: didAddTouchEventHandler()` call this instead of adding it here?

Reply:
Fixed.